WeakAurasOptions/LoadOptions.lua

```diff
@@ -99,7 +99,7 @@ local function setValue(trigger, field, value, multiEntry, entryNumber)
         trigger[field] = { trigger[field] }
       end
     end
-    if value == "" then
+    if value == "" or value == nil then
```

Comment:
Do we need to fixup triggers with holes?

Reply:
migration to fix that added